src/OpenTelemetry/CHANGELOG.md

```diff
@@ -11,15 +11,16 @@
   `CustomProperty`.
   ([#1463](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1463))
 * Remove RentrantExportProcessor as it is not required by spec.
+* `ActivitySourceAdapter.Start` requires `ActivitySource` to update new
```

Comment:
please change to `ActivitySourceAdapter` supports setting ActivitySource for Activities created without ActivitySource.

Reply:
Also add changelog to every instrumentation - HttpInstrumentation sets ActivitySource to activities created outside ActivitySource etc.